### PR TITLE
Adjust production dockerfile

### PR DESCRIPTION
**Pull request by thehabbos007**
## 🌟 What is the purpose of this PR?

This PR adjusts the production dockerfile to remove the hardcoded `kratos.dev.yml` references. Thanks @Alfred-Mountfield for finding this issue :green_heart: 

image

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -22,6 +22,8 @@ HASH_KRATOS_PG_PASSWORD=kratos
 HASH_KRATOS_PG_DEV_DATABASE=dev_kratos
 HASH_KRATOS_PG_TEST_DATABASE=test_kratos
 
+HASH_GRAPH_LOG_LEVEL=info
```
Comment (Alfred-Mountfield) [inline .env:25]: I know we discussed this yesterday, but does this end up used for the development docker compose too? Perhaps we just want to change the defaults in the prod docker compose file, and leave this undefined here

Comment (thehabbos007) [reply .env:25]: It would take precedence over the dockerfile defaults, yes

Comment (Alfred-Mountfield) [reply .env:25]: I think we should probably still have more in-depth logs in trace mode, just not in the prod docker compose

Comment (thehabbos007) [reply .env:25]: In that case I'll revert the commit adding this .env value

Comment (Alfred-Mountfield) [reply .env:25]: Can you also update 
```rml
      RUST_LOG: "${HASH_GRAPH_LOG_LEVEL:-graph=trace,hash_graph=trace,tokio_postgres=debug}"
```
in `docker-compose.prod.yml`

Comment (Alfred-Mountfield) [reply .env:25]: Maybe to 
```yml
      RUST_LOG: "${HASH_GRAPH_LOG_LEVEL:-info}"
```

Comment (Alfred-Mountfield) [reply .env:25]: If that's the right syntax 😅  (not sure)

Comment (thehabbos007) [reply .env]: 169d82b97c245a57a4cd663902b8ff2e0ccdd37e and 4d014730279915ac1601e7aa51293320330c0ef9

**Review by Alfred-Mountfield — APPROVED** [review]

**Pull request merged**